web_widget_number_ux_choice/static/src/scss/number_ux_choice.scss

```diff
@@ -0,0 +1,11 @@
+
+.input_number_ux_choice {
+    width: 71% !important;
```

Comment:
I can't manage to avoid ! important
And I also try flexboxes or to bootstrap it with container but you need to encapsule it in div and I didn't manage to put my field in div without breaking the widget magic..